src/main/java/org/folio/ld/dictionary/model/InstanceMetadata.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.ld.dictionary.model;
+
+import java.util.UUID;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@NoArgsConstructor
+@Accessors(chain = true)
+public class InstanceMetadata {
+  private UUID inventoryId;
+  private UUID srsId;
```

Comment:
Suggest to use simple String instead of UUID

Reply:
Done